source/_docs/create-custom-upstream.md

```diff
@@ -259,11 +259,20 @@ To avoid incompatibilities, you must track Pantheon's corresponding upstream rep
 <div role="tabpanel" class="tab-pane" id="gitlab-https" markdown="1">
 Custom Upstreams from GitLab repositories must be created for you by Pantheon Support.
 
-1. Prepare a GitLab user with at least **Reporter** level permissions for your upstream repository.
+1. Prepare a new GitLab user specifically for Pantheon's use, with at least **Reporter** level permissions for your upstream repository:
 
     ![GitLab reporter permission](/source/docs/assets/images/gitlab-reporter.png)
 
-2. [Contact support](/docs/support) to add the Custom Upstream to your org. You must provide a username/password or OAuth token.
+1. From this new account, [generate an Access Token](https://gitlab.com/profile/personal_access_tokens){.external} with the `API` scope:
+
```

Comment:
Here's an image if it's helpful:

![Screen Shot 2019-03-21 at 9 36 30 PM](https://user-images.githubusercontent.com/5023646/54795152-73af2280-4c21-11e9-86ab-f8eda4006bc3.png)